.eslintrc

```diff
@@ -65,6 +65,14 @@
           "render"
         ]
       }
+    ],
+    "camelcase": [
```

Comment:
@sogame - will change this to not add it here but through `eslint-config-skyscanner` as done here:  https://github.com/Skyscanner/eslint-config-skyscanner/pull/41

Reply:
Done! Updated `eslint-config-skyscanner` instead :-)